pkg/controllers/user/alert/watcher/node.go

```diff
@@ -48,6 +48,9 @@ func (w *NodeWatcher) watch(ctx context.Context, interval time.Duration) {
 }
 
 func (w *NodeWatcher) watchRule() error {
+	if w.alertManager.IsDeploy == false {
```

Comment:
Any reason why we make the checks on per individual methods basis? Can we just chose not to run tasks in some centralized place? based on the alert manager state.